src/Loader.php

```diff
@@ -7,9 +7,9 @@
 namespace Automattic\WooCommerce\Admin;
 
 use \_WP_Dependency;
-use Automattic\WooCommerce\Admin\Features\Onboarding;
```

Comment:
Looks like this was just obsolete already?

Reply:
It was, I think removed a while ago.